tests/Console/Command/AddPrefixCommandIntegrationTest.php

```diff
@@ -283,6 +287,35 @@ private function getNormalizeDisplay(string $display): string
         return implode("\n", $lines);
     }
 
+    private function getNormalizeSeparators(string $display): string
```

Comment:
I think this code could be extracted in a re-usable piece. So I would do like in [humbug/box](https://github.com/humbug/box) and create a [`DisplayNormalizer `](https://github.com/humbug/box/blob/master/tests/Console/DisplayNormalizer.php) helper class. I would then also rename `getNormalizeSeparators()` for `normalizeSeparators()`.

The `if ('\\' === DIRECTORY_SEPARATOR) {` check can also be moved here. This way you always call the `normalizeSeparators` method without caring about the platform